### ISSUE TITLE
running "apatite render" results in dozens of ProjectValidationError

### ISSUE THREAD

**Issue by mcgyver5** [mahmoud/awesome-python-applications#134]
I cloned the repo and installed apatite.

I ran "apatite render" and got dozens of errors like the following:
ProjectValidationError('Wrong key \'date_added\' in ordereddict([(\'name\', \'pip\'), (\'repo_url\', \'https://github.com/pypa/pip\')

rendering the README.md as required for contributions has failed.

**Comment by mahmoud** [comment]
Oh! You're very awesome for wanting to do the rendering yourself. What's happened here is the list grew a feature that the released version of apatite hasn't caught up with. But apatite trunk will work! So you can do what I do and run:

```
$ pip install -e git+git@github.com:mahmoud/apatite.git#egg=apatite
```

To get the newest apatite and try rendering with that. Or, you can just update the YAML and I'll take care of fixes and rendering. :) I'll get a release of apatite out soon in any case!

**Comment by mahmoud** [comment]
fwiw, I cut a new apatite version, so if you `pip install --upgrade apatite`, things should work again. :)

**Comment by mcgyver5** [comment]
ok.  I will do the upgrade.  Thanks for responding!

**Comment by mahmoud** [comment]
Fixed in apatite 20.0.1, thanks again for the report! :)
